hdl21/primitives.py

```diff
@@ -400,6 +405,25 @@ class PhysicalCapacitorParams:
     aliases=["Cap3", "PhyCap3", "CapPhy3", "PhyCapacitor3"],
 )
 
+# For the 4 terminal components, we have
```

Comment:
Same comment as for Sky130: I don't think we should have this as a generic. 

There's too much interpretation to be had in what the "other two" terminals mean.